Updates for Terraform policy docs (#2864)
hashicorp/web-unified-docs: content/terraform-policy/v0.1.x (beta)/docs/policy/write/policies.mdx

## Policy files

> Create a `.policy.hcl` file to store related policies. This file includes configuration for Terraform policy itself. It also contains one or more policy definitions. Each policy definition can target a specific Terraform construct.

Reviewer: ```suggestion
Create a `.policy.hcl` file to store related policies. This file contains configuration for the policy and one or more policy definitions. Each policy definition can target a specific Terraform construct.
```
Simpler?

Terraform policy's configuration language defines blocks to allow you to write policies for Terraform providers, resources, and modules:

- To write policies for Terraform providers, refer to [`provider_policy`](/terraform/policy/reference/policy/provider-policy).
- To write policies for Terraform resources, refer to [`resource_policy`](/terraform/policy/reference/policy/resource-policy).
- To write policies for Terraform modules, refer to [`module_policy`](/terraform/policy/reference/policy/module-policy).

### Arguments and blocks

Like Terraform configuration, policy blocks contain arguments and blocks that allow you to configure the policy. Each policy block type supports various arguments and blocks. Refer to the individual policy block reference pages for a list of supported arguments and blocks.

### Terraform attributes

> You can reference attribute and meta-argument values from Terraform runs in your policies. Use the `attr` prefix to refer to provider or resource attributes, and `meta` to refer to Terraform meta-arguments such as provider and module source and versions. Refer to the individual policy block reference pages for a list of supported attributes and meta-arguments.

Reviewer: ```suggestion
You can reference attribute and meta-argument values from Terraform runs in your policies. Add the `attr` prefix to expressions to reference provider or resource attributes. Add the `meta` prefix to reference Terraform meta-arguments, such as provider and module source and versions. Refer to the individual policy block reference pages for a list of supported attributes and meta-arguments.
```
I would still prefer "Terraform arguments" as opposed to "Terraform meta-arguments" here because "Terraform meta-arguments" might confuse people who are thinking of constructs such as `for_each` and `lifecycle`. But I defer to you.

> You can reference attribute and meta-argument values from Terraform runs in your policies. Reference provider or resource attributes with the `attr.` prefix. When you write policies for resource update operations, reference the prior state of resource attributes with the `prior_attrs.` prefix. Reference Terraform arguments such as provider and module sources with the `meta.` prefix. Refer to the individual policy and test block reference pages for a list of supported attributes and meta-arguments.

Reviewer 2: The prefix is `attrs.`, not `attr.` (used correctly everywhere else in these docs).
